packages/terra-button/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Unreleased
 ------------------
 ### Changed
 * Update react/react-dom peer-dependency to latest stable minor release
+* Reset default margin in Safari. [#471](https://github.com/cerner/terra-clinical/issues/471)
```

Comment:
No need to provide the issue link here.

Reply:
Lets move this under the unreleased heading under the section `### Fixed` to align with our [changelog conventions](https://keepachangelog.com/en/1.0.0/).  When we release we will update with the date and release version

Reply:
Done - https://github.com/cerner/terra-core/pull/2316/commits/8b4ce473766ef3d3bcc2e3a7f8fc71df2ed25756.